test/raygun.messageBuilder_test.js

```diff
@@ -377,6 +377,10 @@ test("filter keys tests", function (t) {
   });
   var message = builder.build();
 
+  // Original object should not be modified
+  t.equal(body.username, "admin@raygun.io");
+  t.equal(body.password, "nice try");
+
```

Comment:
Added to ensure that we do the copy in the `filterKeys` method, i.e. `const _obj = { ...obj } as Record<string, object>;`